internal/task/task/fs/mount.go

```diff
@@ -39,6 +39,9 @@ const (
 	FORMAT_MOUNT_OPTION = "type=bind,source=%s,target=%s,bind-propagation=rshared"
 
 	CLIENT_CONFIG_DELIMITER = "="
+
+	LAYOUT_CURVEFS_TOOLS_CONFIG_SYSTEM_PATH = "/etc/curvefs/tools.conf"
```

Comment:
It's better to define a module level funtion in topology module to reduce redefine, like `GetProjectLayout`.
[see also](https://github.com/opencurve/curveadm/blob/d7e25679c8452b01dba4e00e73caa60896aa1421/internal/configure/topology/dc_get.go#L238)